Patches/Caffe/WindowsDownloadPrebuiltDependencies.cmake

```diff
@@ -0,0 +1,182 @@
+set(DEPENDENCIES_URL_1800_27 "https://github.com/willyd/caffe-builder/releases/download/v1.1.0/libraries_v120_x64_py27_1.1.0.tar.bz2")
```

Comment:
I could, but WindowsDownloadPrebuiltDependencies.cmake (before my changes) comes straight from the source. Adding attribution here doesn't seem to mesh with how we do patches in general. For example, see Patches/Caffe/Dependencies.cmake (the only patched file in Caffe before my changes), which mostly comes straight from the source but doesn't include attribution to BVLC.